cloudcix_primitives/nat_firewall_ns.py

```diff
@@ -1,18 +1,305 @@
 # stdlib
+import json
 from typing import Tuple
 # lib
+from cloudcix.rcc import CHANNEL_SUCCESS, comms_ssh, CONNECTION_ERROR, VALIDATION_ERROR
 # local
+from cloudcix_primitives.utils import load_pod_config, PodnetErrorFormatter, SSHCommsWrapper
 
 
 __all__ = [
     'build',
     'read',
+    'scrub',
 ]
 
+SUCCESS_CODE = 0
 
-def build() -> Tuple[bool, str]:
-    return(False, 'Not Implemted')
+def build(
+        namespace: str,
+        one_to_one: dict,
```

Comment:
```suggestion
        one_to_one: List[Dict[str, str]],
```

`List` and `Dict` can be imported from typing library